src/analysis/AppLauncher.ts

```diff
@@ -218,11 +251,52 @@ const ApplicationLauncher = _.flow(
     });
 
     useEffect(() => {
-      const runtime = getCurrentRuntime(runtimes);
       const runtimeStatus = getConvertedRuntimeStatus(runtime);
 
+      const setupWelder = _.flow(
+        Utils.withBusyState(setBusy),
+        withErrorReporting('Error setting up analysis file syncing')
+      )(async (runtime: Runtime) => {
```

Comment:
main different is that this now takes a runtime and is only run when the runtime exists